### PR TITLE
small change in LLS and COS-Dwarfs

### PR DESCRIPTION
**Pull request by marijana777**
1) Script pyigm_fitlls does not work in all computers, possibly because of python 2 vs python 3 differences. Debugging. 
2) Adding an option to define separate path to COS Dwarfs data (cdir).

### CODE REVIEW

File: pyigm/abssys/lls.py [hunk 1]
```diff
@@ -498,6 +498,7 @@ def tau_multi_lls(wave, all_lls, **kwargs):
     from linetools.analysis import voigt as lav
     #
     all_tau_model = np.zeros(len(wave))
+    all_tau_model = list(np.zeros(len(wave)))  ## need list not array for python3 ?
```
Comment (profxj) [inline pyigm/abssys/lls.py:501]: Please explain why this needs to be a list.

**Comment by profxj** [comment]
When you do a PR, a description is expected.  Please provide one.

File: pyigm/abssys/lls.py (hunk 1 above)
Comment (marijana777) [reply pyigm/abssys/lls.py:501]: If this is not a list, it gives an error AttributeError: 'numpy.ndarray' object has no attribute '_set_unit', and the gui does not fit a LLS. With list it fits a LLS. I am not sure about exact reasons, but I am assuming that this is probably a difference between python 2 and 3.

Comment (profxj) [reply pyigm/abssys/lls.py:501]: There is no way that 

```
all_tau_model = np.zeros(len(wave))
```

will give a _set_unit error.

You must be referring to other code.  
Please expplain.

**Review by profxj — CHANGES_REQUESTED** [review]

File: pyigm/abssys/lls.py (hunk 1 above)
Comment (marijana777) [reply pyigm/abssys/lls.py:501]: The part of the code that is giving that error is
(line 519 in lls.py)   all_tau_model += tau_model

Comment (profxj) [reply pyigm/abssys/lls.py:501]: Then why make the change here?  Might it not break
things elsewhere?

Comment (marijana777) [reply pyigm/abssys/lls.py:501]: I would propose making the change in the same place, and adding at the end of the tau_multi_lls function something that would convert tau_multi_lls to its original type (analogue to all_tau_model = np.asarray(all_tau_model)). Could you let me know what is the original array type? Thanks.

Comment (marijana777) [reply pyigm/abssys/lls.py:501]: I tried adding all_tau_model = np.asarray(all_tau_model) at the end, but the gui was then not able to fit LLS. Also redshift for continuum fitting could not be QSO redshift. I suggest we discuss this more tomorrow, and if you can show me an example how is the gui supposed to work. I would also suggest adding docs on this gui.

Comment (profxj) [reply pyigm/abssys/lls.py:501]: Please do add docs on the GUI.  I'm making an Issue
and assigning it to you.

**Comment by profxj** [comment]
@marijana777 -- Is this PR ready for merging now?

**Comment by marijana777** [comment]
(2) Adding an option to define separate path to COS Dwarfs data (cdir). is ready for merging,
(1) could be ignored if it is solved in another way. Checking

**Comment by marijana777** [comment]
Well...... The last version in master branch that I had was 6f769a3  (6/14/2018). I did git pull, and tried pyigm_fitlls again, and got error  
Traceback (most recent call last):
  File "/home/smailagic/anaconda3/bin/pyigm_fitlls", line 11, in <module>
    load_entry_point('pyigm', 'console_scripts', 'pyigm_fitlls')()
  File "/home/smailagic/pyigm/pyigm/scripts/pyigm_fitlls.py", line 54, in main
    lls_fit_file=pargs.lls_fit_file, norm=args.norm)
AttributeError: 'NoneType' object has no attribute 'norm'

The version in llsdw branch is at least working reasonably well.

**Comment by marijana777** [comment]
Is this PR ready for merging now? - - I would say yes.

**Comment by jnburchett** [comment]
@marijana777 are you waiting on your changes to be accepted? I see one outstanding review from @profxj, but I'm not sure if this is the hold up.  Thanks!

**Comment by marijana777** [comment]
@jnburchett , yes, I am waiting for my changes to be accepted, or for @profxj to let me know if he wants me to change something more. Thanks for asking!

**Review by profxj — APPROVED** [review]
merging

**Pull request merged**